docs/running-workflows/supported-systems/yadage/index.md

```diff
@@ -1,3 +1,23 @@
 # Yadage
 
-The Yadage workflow system originated in particle physics community.
+The Yadage workflow system has been developed in the context of High Energy
+Physics analyses in order to realized re-usable analyses for systematic
```

Comment:
```suggestion
Physics analyses in order to realise re-usable analyses for systematic
```

or perhaps "perform".

I'm sure @tiborsimko has a better feedback for these descriptions.